packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss

```diff
@@ -2,7 +2,8 @@
 
 .activeVisitsContainer {
   background-color: $ui-background;
-  width: 100%;
+  // feedback needed
```

Comment:
Yes. Why 90? Why `vw`?

Reply:
If you see the current widget on the home screen, the width changes according to the content in different pages and hence I wanted to fix the size. If I fix the size to 768px(tab width) then the date divided into 2 lines. I discussed it with JJ as in to what to do, he said, you can help in it and thought of keeping to the screen width and hence `90vw` .

Reply:
Yes, but 90vw does not make much sense, e.g., on mobile. You shouldn't fix the width and you should try to make a progressive design, i.e., one that pretty much *always* works.

So what if the nav bar is shown on the left side? What if the user has a wide screen (like 4k pixels wide)? 90vw does not sound right,.

Reply:
Sure, I get you, let me fix that too.
Thanks!